projects/qt5.cmake

```diff
@@ -212,7 +212,7 @@ function(build_qt5)
     add_custom_target(qt5_build ALL
       COMMENT "Configuration complete...building qt5"
       WORKING_DIRECTORY ${QT5_REPO_PATH}
-      COMMAND make -j2
+      COMMAND make -j4
```

Comment:
This put the build time down to ~50 minutes on my VM so I figured I'd leave it in.